### ISSUE TITLE
is it safe to play after the game gets updated?

### ISSUE THREAD

**Issue by dan993398** [Jire/Charlatano#439]
Just wanted to make sure ..if its safe or not to play with this cheat after the games gets updated?
or should i wait for Jire to update the hack?

**Comment by Jire** [comment]
Yes. No update is needed, and safety is not a concern for updates.

**Comment by dan993398** [comment]
Thanks for the quick reponse.., So u mean no offsets need to be changed even after the update?

**Comment by dan993398** [comment]
sorry but m noob

**Comment by tommy2407** [comment]
@Jire can you check Recoil and Aim setting? I think its broken. I try with bot and it doesn't work. Thank @Jire !

**Comment by dan993398** [comment]
Recoil only works when ACTIVATE_FROM_FIRE_KEY is enabled.. 
it doesn't work with Force Aim Key

**Comment by tommy2407** [comment]
@dan993398 I just checked back, everything was the same

**Comment by Clara1337** [comment]
@tommy2407 My Aimbot work fine .
